packages/valaxy/client/utils/time.ts

```diff
@@ -35,7 +35,12 @@ export function formatDate(date: string | number | Date, formatStr = 'yyyy-MM-dd
     return formatWithTZ(zonedDate, formatStr, { timeZone: options?.timeZone })
   }
   catch (error) {
-    console.error('Error formatting date:', date, error)
+    console.error(
```

Comment:
I suggest that we warn user about invalidity of datetime at build time too. Currently user will only get noticed when they visit a page with a non-formal date, as this console error is only effective at client side.

Reply:
This function will be executed during the build process. Naturally, this warning will appear, which requires the user's attention, but it will not prevent the build from proceeding.

Reply:
`console.error` is a method of the built-in global object `console` in Node.js, and it is not limited to just the browser.